### ISSUE TITLE
Glitch in assigning hotkeys in Gtk version

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#9]
```
What steps will reproduce the problem?
1. Go to Edit > Preferences > Special Hotkeys
2. Clear one of the hotkeys.
3. Try to assign this hotkey again.

What is the expected output? What do you see instead?
AutoKey tells me that the hotkey is already assigned. This is wrong, as I have 
just cleared it. The 
workaround: Clear the hotkey, save the preferences, go to the preferences 
again, assign hotkey.

What version of the product are you using? On what operating system?
0.60.7a, Gtk version
Linux Mint Gloria (based on Ubuntu 9.04)

```

Original issue reported on code.google.com by `alex.bepple` on 19 Nov 2009 at 3:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `cdekter` on 24 Mar 2010 at 11:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `cdekter` on 24 Mar 2010 at 11:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.61.6
```

Original comment by `cdekter` on 25 Mar 2010 at 1:46
- Changed state: **Fixed**
